whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapRelationService.java

```diff
@@ -75,6 +85,39 @@ public List<String> getInetnumRelationPkeys(final String pkey, final RelationTyp
                 .toList();
     }
 
+    public void mapRelationLinks(final RdapObject rdapResponse, final String requestUrl){
+        if (requestUrl == null || requestUrl.contains("rirSearch1")){
```

Comment:
change this to "rirSearch1".contains(requestUrl). Better to use enum RdapConformance

Reply:
good catch, but now I did refactor this.
This not longer applies